crowdloan/polkadot-dev.json

```
@@ -224,5 +224,13 @@
     "description": "OriginTrail is the world’s first Decentralized Knowledge Graph️",
     "website": "https://parachain.origintrail.io/",
     "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/OriginTrail.svg"
+  },
+  {
+    "paraid": "2007",
+    "name": "Totem",
```

```suggestion
    "name": "Totem Kapex",
```
I think it will better, based on fact that in Polkadot.js/apps is used "Kapex" for this network and looks like users know that name better.